library/matomo

```diff
@@ -2,17 +2,17 @@
 Maintainers: Pierre Ozoux <pierre@piwik.org> (@pierreozoux)
 GitRepo: https://github.com/matomo-org/docker.git
 
-Tags: 3.14.0-apache, 3.14-apache, 3-apache, apache, 3.14.0, 3.14, 3, latest
+Tags: 3.14.1-apache, 3.14-apache, 3-apache, 3.14.1, 3.14, 3
```

Comment:
Was the loss of `apache` and `latest` here intentional?

Reply:
No
https://github.com/docker-library/official-images/pull/8935